tools/starman.psgi

```diff
@@ -27,6 +28,11 @@ builder {
     enable match_if path(qr!.+\.(css|js|png|ico|jp(e)?g|gif)$!),
         'ConditionalGET';
 
+    enable "Plack::Middleware::Pod",
```

Comment:
@ylavoie above, the Plack::Middleware::Pod module is required conditionally, yet the 'enable' statement here seems to be executed unconditionally. Does that work?